src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -161,3 +161,55 @@ def snapshot(self) -> Iterable[ClusterInfo]:
     def _try_fetch(self) -> Iterable[ClusterInfo]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield ClusterInfo(*row)
+
+
+@dataclass
+class PolicyInfo:
+    policy_id: str
+    cluster_id: str
```

Comment:
Policy applies to multiple clusters. This field doesn't belong here.
```suggestion
```

Reply:
Removed the cluster Id